apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -72,7 +114,7 @@ const Auth: React.FC = () => {
 					</div>
 
 					<h1 className={styles["form-header__text"]}>
-						{pathname === AppRoute.SIGN_IN ? "sign in" : "create an account"}
+						{authRouteToHeader[pathname as ValueOf<RoutesWithHeader>]}
```

Comment:
Why not ValueOf<typeof AppRoute>? I prefer using AppRoute, instead of creating a new type

Reply:
We have a lot of routes in app route that do not have an associated header in this case, and the amount of them will only increase in future.

Reply:
If I do ValueOf< typeOf AppRoute >, typescript requires me to provide an entry for every route to the map. Creating RoutesWithHeader type just saves a lot of headache in the future, it is also gives a more descriptive type hint when working with AuthRouteToHeader since it gives the idea that only certain keys yield actual result